src/App/Model.elm

```diff
@@ -80,19 +80,18 @@ type alias Flags =
 
 
 type alias Model =
-    { currentPage : Page
-    , flags : Flags
-    , pages :
-        { home :
-            { promotedArticles : WebData (List Article)
-            , promotedRecipes : WebData (List Recipe)
-            , recipes : WebData (List Recipe)
-            }
-        , articles : WebData (List Article)
-        , recipes : Dict String (WebData (List Recipe))
-        , recipe : { recipe : WebData Recipe, recipes : WebData (List Recipe) }
-        , contact : ContactForm
+    { flags : Flags
+    , currentPage : Page
+    , pageHome :
+        { promotedArticles : WebData (List Article)
+        , promotedRecipes : WebData (List Recipe)
+        , recipes : WebData (List Recipe)
         }
+    , pageAboutUs : {}
+    , pageRecipes : Dict String (WebData (List Recipe))
```

Comment:
This string is for the category name, but I agree having its own type is not a bad idea.